src/Cassandra/HostConnectionPool.cs

```diff
@@ -218,6 +218,8 @@ internal void AttemptReconnection()
                 }
                 else
                 {
+                    if (t.Exception != null)
+                        Logger.Error("Error during reconnection attempt", t.Exception);
                     _poolModificationSemaphore.Release();
                     Logger.Info("Reconnection attempt to host {0} failed", _host.Address);
                     _host.SetDown();
```

Comment:
Should handle `t.Exception` here also? `AttemptReconnection()` is `void` and it wont be possible to do it anywhere else.